src/styles/shared/_buttons.scss

```diff
@@ -201,6 +201,7 @@
   border-color: rgba($outline-color, 0.4);
   box-shadow: none;
   color: darken($outline-color, 20%);
+  @include recolor-icon(darken($outline-color, 20%));
```

Comment:
Right now, a destructive outline icon button will have a bright red icon fill and darkened text. This isn't a variant represented in the style guide or in the UI kit, so I think this was just an oversight.

Reply:
On the UI kit, the icon on the basic button is two tones lighter `Ink lighter` while the text is `Ink`, so I think the example on "Before" is actually correct.
Aso, the outline seems lighter than what's in the style guide.
<img width="82" alt="Screen Shot 2019-12-06 at 9 42 26 AM" src="https://user-images.githubusercontent.com/5349023/70331081-bdba5180-180c-11ea-9014-44c1d10655cf.png">

Reply:
Hmmm, those screenshots are from the style guide, is your screenshot an outline button or a monochrome button?

Reply:
My screenshot is from the outline monochrome button (style guide). I'm sorry for the late reply!